### PR TITLE
add Append command

### PR DESCRIPTION
**Pull request by aureocarneiro**
Add Append command for SEQ table usage.

### CODE REVIEW

File: pandablocks/commands.py [hunk 1]
```diff
@@ -293,6 +294,30 @@ def execute(self) -> ExchangeGenerator[None]:
...
+class Append(Command[None]):
+    """Append the value of a table field.
+
+    Args:
+        field: The field, attribute, or star command to append
+        value: The value, list of strings, to append
+
+    For example::
+
+        Put("SEQ1.TABLE", ["1048576", "0", "1000", "1000"])
```
Comment (AlexanderWells-diamond) [inline pandablocks/commands.py:307]:
```suggestion
        Append("SEQ1.TABLE", ["1048576", "0", "1000", "1000"])
```

**Review by AlexanderWells-diamond — CHANGES_REQUESTED** [review]
Thank you for the PR. It looks useful. 

One required change as per my suggestion.

Could you please add one-or-more tests? Similar, easily adaptable ones are for example `test_connect_put_multi_line()` in `tests/test_pandablocks.py`. Any of the `test_connect_put_*` tests should show you examples.

File: pandablocks/commands.py [hunk 2]
```diff
@@ -293,6 +294,30 @@ def execute(self) -> ExchangeGenerator[None]:
...
+    """
+
+    field: str
+    value: Union[List[str]] = ["",]
```
Comment (coretl) [inline pandablocks/commands.py:311]: As this is only for putting to tables, which are multiline, we can simplify the type and make it required...

```suggestion
    value: List[str]
```

File: pandablocks/commands.py [hunk 3]
```diff
@@ -293,6 +294,30 @@ def execute(self) -> ExchangeGenerator[None]:
...
+
+    def execute(self) -> ExchangeGenerator[None]:
+        if isinstance(self.value, list):
+            # Multiline table with blank line to terminate
+            ex = Exchange([f"{self.field}<<"] + self.value + [""])
+            yield ex
+            assert ex.line == "OK"
```
Comment (coretl) [inline pandablocks/commands.py:318]: ...and then we don't need the isinstance

```suggestion
        # Multiline table with blank line to terminate
        ex = Exchange([f"{self.field}<<"] + self.value + [""])
        yield ex
        assert ex.line == "OK"
```

**Review by coretl — COMMENTED** [review]
Thanks for the contribution! I've made a couple of suggestions to simplify. I'll second Alex's request for a test if you have a chance...

**Pull request merged**